ikasaneip/platform/ikasan-setup/src/main/resources/db-changelog-2.1.0.xml

```diff
@@ -25,7 +25,6 @@
     <include file="db/index/db-changeLog-createIndexFlowModuleId.xml" relativeToChangelogFile="true" />
     <include file="db/index/db-changeLog-createIndexIkasanModuleServerId.xml" relativeToChangelogFile="true" />
     <include file="db/index/db-changeLog-createIndexMetricEventExpiry.xml" relativeToChangelogFile="true" />
-    <include file="db/index/db-changeLog-createIndexFTTransactionalResourceCommandXid_Id.xml" relativeToChangelogFile="true" />
     <include file="db/index/db-changeLog-createIndexMetricEventEventId.xml" relativeToChangelogFile="true" />
```

Comment:
How come this is removed?

Reply:
this somehow made it into the wrong context. This is only relevant for modules that require the ftp and sftp tables.